frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -22,6 +22,7 @@ import { PersistentVolumeClaimModel } from '../models';
 
 const { common, ExpandPVC } = Kebab.factory;
 const menuActions = [
+  ...Kebab.getExtensionsActionsForOCS(PersistentVolumeClaimModel),
   ExpandPVC,
   ...Kebab.getExtensionsActionsForKind(PersistentVolumeClaimModel),
```

Comment:
why can't we use this extension?

Reply:
This extension is being used across other views as well. Adding the check here will re-evaluate for other views kebab menus as well. Hence, added a separate function which will just be executed for pvc view.